### DISCUSSION TITLE
Mantling collision settings

### DISCUSSION THREAD

**Discussion by emrehan98** [Sixze/ALS-Refactored#17] [Ideas]
Hey there. I have a suggestion.

Sorry if I missed but I couldn't find collision settings for mantling. It would be nice to have an option to change collision settings within "Mantling Settings". 

I have actors attached to my character, if I start mantling it infidently climbs attached actors so I had to disable mantling for now. If you could give an option withing Mantling Settings it would be amazing.

**Comment by Sixze** [comment]
Good idea.

**Comment by Sixze** [comment]
Check e49cf5d4850b7c8876a9bbdbbb24177294401cb9 commit.

**Reply by emrehan98** [reply]
Much better, thank you for taking your time!
